packages/bot/src/services/api/server_time.js

```diff
@@ -0,0 +1,32 @@
+class PromiseClass {
+    constructor() {
+        this.promise = new Promise((resolve, reject) => {
+            this.reject  = reject;
+            this.resolve = resolve;
+        });
+    }
+}
```

Comment:
This is duplicated, please see `deriv-app/packages/bot/src/utils/pending-promise.js`